### DISCUSSION TITLE
doubt about threshold for grain reconstruction

### DISCUSSION THREAD

**Discussion by alfagm** [mtex-toolbox/mtex#917] [Ask Anything]
Hi,
first of all, sorry if my question is well stablished knowledge, I am a beginner.
I am trying to improve the reconstruction of my grains and removal of my wild spikes of my EBSD data. I found this line of code in several scripts:

ebsd(grains(grains.grainSize<=40)) = [];

and I was wondering about the number 40. Is this number a well established threshold? What is it based on?

As it is now, I loose considerable amounts of my smaller well-indexed grains and I get better results by applying a lower threshold (for instance, 20). However, I am not sure if to do so because I have seen this number 40 in different scripts and so I wonder if it is some kind of standard.
Can anyone clarify me this?
Thank you very much for any given help! 

Best,

Marina

**Comment by kilir** [comment]
Hi,
grain2d/grainSize is simply the number of measurement points in a grain.

The number you took it's just a number used by whoever wrote the script and who had a reason to get rid of the ebsd which belongs to grains with less than 40 measurement points.

Hope that helps.

Cheers,
Rüdiger

**Reply by alfagm** [reply]
Thanks! That is what I thought at the beginning but I wanted to check just in case.
Best,
Marina
